src/test/java/test/listeners/ExecutionListenerAndSuiteListenerTest.java

```diff
@@ -0,0 +1,38 @@
+package test.listeners;
+
+import static org.testng.Assert.assertEquals;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.testng.TestNG;
+import org.testng.TestNGException;
+import org.testng.annotations.Test;
+import org.testng.xml.Parser;
+import org.testng.xml.XmlSuite;
+import org.xml.sax.SAXException;
+
+
+public class ExecutionListenerAndSuiteListenerTest {
+
+    @Test
+    public void executionListenerAndSuiteListenerTest() {
+        String suiteFile = "src/test/resources/executionlistenersingletoncheck/parent.xml";
```

Comment:
Use SimpleBaseTest#getPathToResource
https://github.com/cbeust/testng/blob/f8bd4f43be13b6ffad1741c36604d91510c37321/src/test/java/test/SimpleBaseTest.java#L269

Reply:
will do it.